README.md

```diff
@@ -1,24 +1,174 @@
-# README
+<a name="readme-top"></a>
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you guys make your own logos of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍